widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -8311,3 +8315,15 @@ body.bootstrap .modal.confirmationDialog .dialog div.body {
 .is-hidden {
   display: none;
 }
+
+.users-roles-list,
+.users-roles-selection-list {
+  width: calc(200px - 2 * var(--focus-ring-padding)) !important;
+  margin: var(--focus-ring-padding);
+}
+
+#adminCatTree div[role='tree']:focus-visible {
+  z-index: 0 !important;
+  outline: var(--focus-ring) !important;
+}
```

Comment:
AFAICT, this could be written generally for XUL's Tree control by targeting the `.tree-scroll-panel` class, as is already done elsewhere (`.tree-scroll-panel div[role='tree']:focus-visible`). I also question the need for the `role` selection. AFAICT, could equally be `.tree-scroll-panel > * > :focus-visible`.

Reply:
addressed